hardhat/test/unit/01_Storage.t.js

```diff
@@ -0,0 +1,688 @@
+const {
+  loadFixture,
+} = require("@nomicfoundation/hardhat-toolbox/network-helpers");
+const { expect } = require("chai");
+const hre = require("hardhat");
+const { ethers } = hre;
+const { constants, mocked, fixtures } = require("../helper_fixture");
+const {
+  getRandomAddress,
+  getZeroAddress,
+  getRandomBytes,
+} = require("../../utils/util");
+
+describe("Storage", function () {
+  describe("Storage deployment", function () {
+    it("should store the admin at deployment", async function () {
+      const { storage, owner } = await loadFixture(fixtures.deployStorage);
+      expect(await storage.getOperator("admin")).to.equal(owner.address);
+    });
+
+    it("should have a default value for a non init key->address", async function () {
+      const { storage } = await loadFixture(fixtures.deployStorage);
+      const key = await storage["getKey(string)"]("test");
+      const result = await storage.getAddr(key);
+      expect(result).to.equal(getZeroAddress());
+    });
+    it("should have a default value for a non init key->uint", async function () {
+      const { storage } = await loadFixture(fixtures.deployStorage);
+      const key = await storage["getKey(string)"]("test");
+      const result = await storage.getUint(key);
+      expect(result).to.equal(0);
+    });
+    // @todo => complete init tests
```

Comment:
`// @todo => complete init tests`

is this still relevant?

Reply:
Yes but the comment is not necessary.

Removed